src/api/app/views/webui2/webui/request/_show_side_links.html.haml

```diff
@@ -0,0 +1,47 @@
+%ul.list-unstyled
+  %li
+    %i.fas.fa-user.text-success
+    Created by
+    = user_with_realname_and_icon(request_creator, no_icon: true, short: true)
+    = fuzzy_time(request_created_at)
+  %li
+    %i.fa.fa-info-circle.text-info
+    In state
+    = link_to(request_state, { anchor: 'request_history' }, style: "color: #{request_state_color(request_state)};")
+  - unless package_maintainers.empty?
+    %li
+      %i.fas.fa-eye.text-info{ title: 'Look into' }
```

Comment:
I think the title `'Look into'` doesn't bring value here. I would remove it.

Reply:
look, eye, get it? at the obs-pattern we have the magnifying glass for show. The eye was already there and I do not have a strong opinion against it.. any idea @dmarcoux

Reply:
Why not the `fa-info-circle`?